docs/integrations/amazon-aws/ses.md

```diff
@@ -383,7 +383,7 @@ See the details of bounced notifications by email addresses, domains, bounce typ
 
 See information about complaints (a complaint occurs when a recipient reports that they don't want to receive an email), including the top email addresses, email domains, and UserAgents associated with complaints; and the sending AccountId, AWS region, SourceIP, and Identity associated with complaints.
```

Comment:
```suggestion
See information about complaints (a complaint occurs when a recipient reports that he doesn't want to receive an email), including the top email addresses, email domains, and user agents associated with complaints and the sender's AccountId, AWS region, SourceIP, and Identity associated with complaints.
```

Reply:
Did you mean to write "sender's" instead of sending?

Reply:
Yes, good catch, updating this.